.goreleaser.yml

```diff
@@ -82,6 +76,7 @@ snapshot:
 
 release:
   name_template: "v{{.Version}}"
+  prerelease: auto
```

Comment:
This marks GitHub releases as pre-release if a pre-suffix is detected in the tag.